### PR TITLE
fix(chat): add flag for updating new conversation (Issue #2267)

### PR DESCRIPTION
**Pull request by Gimir**
**Description:**

- add isNewConversationUpdating for newly created conversations

Issues:

- Issue #2267 

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by Gimir** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2607.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2607.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat/src/store/conversations/conversations.reducers.ts [hunk 1]
```diff
@@ -80,6 +81,9 @@ export const conversationsSlice = createSlice({
       if (state.isMessageSending) {
         state.isMessageSending = false;
       }
+      if (state.isNewConversationUpdating) {
+        state.isNewConversationUpdating = false;
+      }
     },
     saveConversationFail: (state, { payload }: PayloadAction<Conversation>) => {
```
Comment (IlyaBondar) [inline apps/chat/src/store/conversations/conversations.reducers.ts:84]: need to add
`state.isNewConversationUpdating = false;`
into `saveConversationFail`

**Comment by Gimir** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2607.nightly-test.deltixhub.io
>E2E tests status: success

**Review by IlyaBondar — APPROVED** [review]

**Pull request merged**